### ISSUE TITLE
Implement refund of ticket upto 2 weeks before party

### ISSUE THREAD

**Issue by petterroea** [InfectedLan/InfectedAPI#83]
Not a need or anything, but warbo came with the idea, so i'l pop it here

**Comment by halvors** [comment]
Added basic isRefundable() to ticket in commit b23c0003a3affc72b88d7b7aa263dcd44c88b55a

**Comment by halvors** [comment]
Now we only need the API for actually refunding tickets. See issue #69.

**Comment by petterroea** [comment]
Yep. I'l look at making a TransactionHandler that we can extend with a PaypalTransatcionHandler. What do you think? It would make it easier to support other payment methods

**Comment by halvors** [comment]
Um. What you mean is that we make an general TransactionHandler that is parent to PaypalTransactionHandler? Also, why don't add BitPay? :dancer:

**Comment by petterroea** [comment]
Yes, that is what i mean.

BitPay would be cool, but we have to go through warbo to add that. I don't think the leadership would accept :cry:

**Comment by halvors** [comment]
@petterroea Do you think that we should postpone this or should we do this before the ticket sale opens 25. August?

**Comment by halvors** [comment]
This is needed for 0.3?

**Comment by petterroea** [comment]
No, almost nobody asks for a refund. Let's postpone.

**Comment by halvors** [comment]
Postponing until 0.4.
